### DISCUSSION TITLE
Does wxMaxima give the possibility to either rearrange the menubar or modify the menubar's entries?

### DISCUSSION THREAD

**Discussion by Rayon340** [wxMaxima-developers/wxmaxima#1874] [Q&A]
Dear everyone (of course), good morning. Do you know if wxMaxima gives the possibility to either rearrange the menubar or modify the menubar's entries? For example, maybe I could want the "Numeric" menu to be before the "Plot" menu. Is it also possible to either modify existing menubar's entries or create a custom menu with our preferred functions? I mean something like what LibreOffice allows us to do.

Thanks for the asnwer. 

Best regards.

**Comment by gunterkoenigsmann** [comment]
wxWidgets doesn't really provide that functionality. In theory it doesn't completely hinder us from implementing that functionality ourselves. But that would involve a complete rewrite of the code that generates the menus...

**Comment by gunterkoenigsmann** [comment]
If that were possible I would not being opposed to wxWidgets providing said functionality: That would make it available for thousands of applications at once.

**Comment by Rayon340** [comment]
Thank you for your answers. I am not able to help the developers of wxWidgets to implement that functionality, but I think that it could be useful to have the possibility to modify menus. I do not know what The Document Foundation team uses to develop LibreOffice, but LibreOffice has an amazing possibility of menu personalization.
